akka-docs/_sphinx/themes/akka/static/docs.css

```diff
@@ -226,13 +226,6 @@ p.caption { width: 80%; text-align: justify; font-size: 0.95em; font-style: ital
   text-decoration: none;
 }
 
-.algolia-autocomplete {
-  display: block;
-  width: 100%;
-  height: 100%;
-  top: -5px;
-  left: 10px;
-}
```

Comment:
It seems IE doesn't render the algolia search box. So it is a plain &lt;input&gt; in IE. 

So moving top -5px to breadcrumb input. That consistently works across browsers.